src/Contracts/Parameters/CombinedCheckerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Neomerx\JsonApi\Contracts\Parameters;
+
+/**
+ * Interface CombinedCheckerInterface
+ * @package Neomerx\JsonApi
+ */
+interface CombinedCheckerInterface extends HeaderCheckerInterface, ParameterCheckerInterface
```

Comment:
Yep I like `ParametersCheckerInterface` = `HeadersCheckerInterface` + `QueryCheckerInterface`. Will make that change.